frontend/packages/dev-console/src/components/import/image-search/ImageSearch.tsx

```diff
@@ -62,6 +62,7 @@ const ImageSearch: React.FC = () => {
           setFieldValue('isi.status', status);
           setFieldValue('isi.ports', ports);
           setFieldValue('image.ports', ports);
+          setFieldValue('image.tag', tag);
```

Comment:
Any specific reason for passing tags fetched from imagestream to `image.tag` here, can we use isi tag only for labels in create service/routes.

Ideally, both would be same but it would be good to have them separate isi tag would be one generated as part of imageStream and image tag would be one provided by user( we may need this later as of now we are not fetching user entered tags)

Reply:
If you see the code of shared submit utils, I have made it usable for both forms. So there are some properties of git import form data that are not available in deploy image form data and vice versa, for such properties I am using `get`. Before I did, if `image` exists in form data then its tag will be used otherwise `isi.tag` will be used, but since `image` is present in both the form data this was not the right way to get the tag because if we are on deploy image form in this case tag will always be empty since we are not  setting `image.tag` anywhere. That is why I had to set it. This `image` and `isi` thing needs to be refactored. I planned to do it in this PR but it turned out that it will need some investigation to refactor it and not break the code anywhere else as both are being used at different places.